saritasa_invocations/mypy.py

```diff
@@ -0,0 +1,15 @@
+import invoke
+
+from . import _config, printing, python
+
+
+@invoke.task
+def run(
+    context: invoke.Context,
+    path: str = ".",
+    params: str = "",
+) -> None:
+    """Run mypy in `path` with `params`."""
+    printing.print_success(f"Mypy running with {params} in {path}")
```

Comment:
Let's do something like to highlight params and path
```suggestion
    printing.print_success(f"Mypy running with '{params}' in '{path}'")
```